src/web/mjs/connectors/Delitoon.mjs

```diff
@@ -0,0 +1,66 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Delitoon extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'delitoon';
+        super.label = 'Delitoon';
+        this.tags = [ 'webtoon', 'french' ];
+        this.url = 'https://www.delitoon.com';
+        this.links = {
+            login: 'https://www.delitoon.com/connexion'
+        };
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.serie-page div.container div.informations-part1.detail div div.bloc-left h2');
+        return new Manga(this, uri.pathname, data[0].textContent.trim());
+    }
+    
+    async _getMangas() {
+        let mangaList = [];
+        let genres = ['Romance','BL','Drama','Sentimental','Historique','Slice%20of%20Life','Fantastique','Comédie','Thriller','Action','Aventure','SF'];
+        for(let page = 0; page < genres.length; page++) {
+            let mangas = await this._getMangasFromPage(genres[page]);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(this.url + '/genres/' + page, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.genres-page div.content ul li');
+        return data.filter(element => element.querySelector('h3'))
+            .map(element => {
+                let title = element.querySelector('div.container-infos h3').textContent.trim();
+                let link = element.querySelector('li a');
+                return {
+                    id: this.getRootRelativeOrAbsoluteLink(link, request.url),
+                    title: title
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(this.url + manga.id, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.container div.episodes div.title ul#tab1 li');
```

Comment:
May consider to use the approach with `closest()` as described in the comment for mangas

Reply:
Suggestion did not work, link and title are not nested, they are in separate branches in the DOM